aspnetcore/toc.md

```diff
@@ -229,85 +207,111 @@
-### [Get started with Blob storage and Visual Studio Connected Services](https://azure.microsoft.com/documentation/articles/vs-storage-aspnet5-getting-started-blobs/)
-### [Get Started with Queue Storage and Visual Studio Connected Services](https://azure.microsoft.com/documentation/articles/vs-storage-aspnet5-getting-started-queues/)
-### [Get Started with Table Storage and Visual Studio Connected Services](https://azure.microsoft.com/documentation/articles/vs-storage-aspnet5-getting-started-tables/)
+# Data access
+## Tutorials
+### EF Core with Razor Pages
+#### [Overview](xref:data/ef-rp/index)
+#### [Get started](xref:data/ef-rp/intro)
+#### [Create, Read, Update, and Delete](xref:data/ef-rp/crud)
+#### [Sort, filter, page, and group](xref:data/ef-rp/sort-filter-page)
+#### [Migrations](xref:data/ef-rp/migrations)
+#### [Create a complex data model](xref:data/ef-rp/complex-data-model)
+#### [Read related data](xref:data/ef-rp/read-related-data)
+#### [Update related data](xref:data/ef-rp/update-related-data)
+#### [Handle concurrency conflicts](xref:data/ef-rp/concurrency)
+### [EF Core with MVC, new DB](/ef/core/get-started/aspnetcore/new-db)
+### [EF Core with MVC, existing DB](/ef/core/get-started/aspnetcore/existing-db)
+### EF Core with MVC, long tutorial
+#### [Overview](xref:data/ef-mvc/index)
+#### [Get started](xref:data/ef-mvc/intro)
+#### [Create, Read, Update, and Delete](xref:data/ef-mvc/crud)
+#### [Sort, filter, page, and group](xref:data/ef-mvc/sort-filter-page)
+#### [Migrations](xref:data/ef-mvc/migrations)
+#### [Create a complex data model](xref:data/ef-mvc/complex-data-model)
+#### [Read related data](xref:data/ef-mvc/read-related-data)
+#### [Update related data](xref:data/ef-mvc/update-related-data)
+#### [Handle concurrency conflicts](xref:data/ef-mvc/concurrency)
+#### [Inheritance](xref:data/ef-mvc/inheritance)
+#### [Advanced topics](xref:data/ef-mvc/advanced)
+## [EF 6 with ASP.NET Core](xref:data/entity-framework-6)
+## Azure Storage with Visual Studio
+### [Connected Services](/azure/vs-azure-tools-connected-services-storage)
+### [Blob storage](/azure/vs-storage-aspnet5-getting-started-blobs/)
+### [Queue storage](/azure/vs-storage-aspnet5-getting-started-queues/)
+### [Table storage](/azure/vs-storage-aspnet5-getting-started-tables/)
+
+# Client-side development
+## [Overview](xref:client-side/index)
+## [Gulp](xref:client-side/using-gulp)
+## [Grunt](xref:client-side/using-grunt)
+## LibMan
+### [Overview](xref:client-side/libman/index)
+### [CLI](xref:client-side/libman/libman-cli)
+### [Visual Studio](xref:client-side/libman/libman-vs)
+## [Bower](xref:client-side/bower)
+## [LESS, Sass, and Font Awesome](xref:client-side/less-sass-fa)
+## [Bundle and minify](xref:client-side/bundling-and-minification)
+## [Browser Link](xref:client-side/using-browserlink)
+## [JavaScriptServices for SPAs](xref:client-side/spa-services)
```

Comment:
2 suggestions:
1. Rename the "SPA project templates" ToC node to "Single Page Apps".
1. Make this JavaScriptServices link a child of the "Single Page Apps" ToC node.